setup.py

```diff
@@ -46,6 +47,9 @@
     # temporarily breaking the builds, remove when fixed
     "sqlalchemy>=1.3,<1.4",
     "flask-sqlalchemy>=2.4,<2.5",
+    'Flask>=1.0.4,<2.0',
```

Comment:
Isn't is enough to simply have `"invenio-base>=1.2.4"`?

Reply:
unfortunately no, in the tests it was still resolved as 2.0.0